<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSMainWidget</class>
 <widget class="QWidget" name="VSMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIMPL Visualization Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <widget class="VSViewWidget" name="viewWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="menuWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="menuLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="filterBtnWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="filterBtnLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="clipBtn">
           <property name="toolTip">
            <string>Clip</string>
           </property>
           <property name="statusTip">
            <string>Clip</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/VtkSIMPL.qrc">
             <normaloff>:/icons/pqClip.png</normaloff>:/icons/pqClip.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sliceBtn">
           <property name="toolTip">
            <string>Slice</string>
           </property>
           <property name="statusTip">
            <string>Slice</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/VtkSIMPL.qrc">
             <normaloff>:/icons/pqSlice.png</normaloff>:/icons/pqSlice.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="maskBtn">
           <property name="toolTip">
            <string>Mask</string>
           </property>
           <property name="statusTip">
            <string>Mask</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/VtkSIMPL.qrc">
             <normaloff>:/icons/pqThreshold.png</normaloff>:/icons/pqThreshold.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>547</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="cameraAxisWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="cameraAxisLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="cameraXpBtn">
           <property name="toolTip">
            <string extracomment="X+">Set view direction to +X</string>
           </property>
           <property name="statusTip">
            <string>Set view direction to +X</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/VtkSIMPL.qrc">
             <normaloff>:/icons/pqXplus.png</normaloff>:/icons/pqXplus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cameraXmBtn">
           <property name="toolTip">
            <string extracomment="X-">Set view direction to -X</string>
           </property>
           <property name="statusTip">
            <string>Set view direction to -X</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/VtkSIMPL.qrc">
             <normaloff>:/icons/pqXminus.png</normaloff>:/icons/pqXminus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cameraYpBtn">
           <property name="toolTip">
            <string extracomment="Y+">Set view direction to +Y</string>
           </property>
           <property name="statusTip">
            <string>Set view direction to +Y</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/VtkSIMPL.qrc">
             <normaloff>:/icons/pqYplus.png</normaloff>:/icons/pqYplus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cameraYmBtn">
           <property name="toolTip">
            <string extracomment="Y-">Set view direction to -Y</string>
           </property>
           <property name="statusTip">
            <string>Set view direction to -Y</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/VtkSIMPL.qrc">
             <normaloff>:/icons/pqYminus.png</normaloff>:/icons/pqYminus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cameraZpBtn">
           <property name="toolTip">
            <string extracomment="Z+">Set view direction to +Z</string>
           </property>
           <property name="statusTip">
            <string>Set view direction to +Z</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/VtkSIMPL.qrc">
             <normaloff>:/icons/pqZplus.png</normaloff>:/icons/pqZplus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cameraZmBtn">
           <property name="toolTip">
            <string extracomment="Z-">Set view direction to -Z</string>
           </property>
           <property name="statusTip">
            <string>Set view direction to -Z</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/VtkSIMPL.qrc">
             <normaloff>:/icons/pqZminus.png</normaloff>:/icons/pqZminus.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VSViewWidget</class>
   <extends>QWidget</extends>
   <header>VSViewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources/VtkSIMPL.qrc"/>
 </resources>
 <connections/>
</ui>
